### PR TITLE
Version

### PR DESCRIPTION
**Pull request by evans22j**
## Implement the following:
- Create a repository and a branch.
- Add HTML,CSS files
- Add toolbar
- Add headline section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        <ul class="nav-list">
+          <li><a href="#portfolio">Portfolio</a></li>
+          <li><a href="#about">About</a></li>
+          <li><a href="#contact">Contact</a></li>
+        </ul>
+        <div class="menu">
+          <i class="bx bx-menu"></i>
+        </div>
+        <div class="close">
+          <img src="Enabled.png" alt="">
```
Comment (Hamzaoutdoors) [inline index.html]: - [ ] Kindly add a text to the alt attribute

File: index.html [hunk 2]
```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>My portfolio</title>
+  <link rel="stylesheet" href="style.css" type="text/css">
```
Comment (Hamzaoutdoors) [inline index.html]: - [ ] I believe that you missed an `s` in `href="style.css"` what do you think, it should be `styles.css`, ^_^

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+html {
+  overflow-x: hidden;
+}
+
+h1 {
+  text-align: left;
+  margin: 149px 0 10px 26px;
+}
+
+.white {
```
Comment (Hamzaoutdoors) [inline styles.css:18]: - [ ] the logo should be in left corner and the nav hamburger should be in the right corner of your header, Kindly try to stick to the design as much as possible and fix that 👍🏼 

real

**Review by Hamzaoutdoors — CHANGES_REQUESTED** [review]
Hey @evans22j, I am **Hamza** your reviewer for this PR 😸 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
✔️ The repo looks professional
✔️ Git/GitHub flow best practices have been followed
✔️ There are no linter errors
✔️ You have a descriptive README

## Required Changes ♻️
- [ ] Please make sure you add meaningful messages to your commits, `Style Update` is very generic and if you find a bug you won't be able to trace down what commit caused it because all the commit messages are the same. Keep this in mind for your future projects as it is a good practice.

- [ ] Next time give your branch a meaningful title, i.e. `mobile-version`, `project-setup`

_See the inline comments below for more required changes 😃_  

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
**N.A**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lordkaito — APPROVED** [review]
Hi @evans22j, 



### Good job


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**